apps/server/apis/grpcapi/external.go

```diff
@@ -1237,6 +1238,55 @@ func (s *ExternalServer) AppRegisterReject(ctx context.Context, req *protos.AppR
 	return s.uibffPostRequest("/v1/app/register/reject", req)
 }
 
+func (s *ExternalServer) GetWasm(ctx context.Context, req *protos.GetWasmRequest) (*protos.GetWasmResponse, error) {
+	if err := validate.GetWasmRequest(req); err != nil {
+		return nil, errors.Wrap(err, "unable to validate GetWasm request")
+	}
+
+	// TODO: Attempt to fetch WasmEntry
+
+	return nil, status.Errorf(codes.Unimplemented, "method GetWasm not implemented")
+}
+
+func (s *ExternalServer) GetAllWasm(ctx context.Context, req *protos.GetAllWasmRequest) (*protos.GetAllWasmResponse, error) {
+	if err := validate.GetAllWasmRequest(req); err != nil {
+		return nil, errors.Wrap(err, "unable to validate GetAllWasm request")
+	}
+
+	// TODO: Attempt to get ALL Wasm entries
```

Comment:
Oh I guess I didn't implement the handlers but that should be trivial now as all of the Store methods have been implemented.